LICENSE-BSD

```diff
@@ -0,0 +1,24 @@
+Redistribution and use in source and binary forms, with or without
```

Comment:
Shouldn't this have copyright info like `Copyright 2018 MaidSafe.net limited.` at the start? (See https://opensource.org/licenses/BSD-3-Clause)

Same for the other copy of this in the -sys lib.

Reply:
@NickLambert thoughts on the above please?

Reply:
Yes it should. If you approve this, I'll raise a PR to ammend it @Fraser999